.github/workflows/cmake.yml

```diff
@@ -61,7 +47,7 @@ jobs:
         environment-file: environment-bnb.yml
         use-only-tar-bz2: false
         auto-activate-base: true
-        python-version: ${{ matrix.python-version }}
+        python-version: "3.10"
```

Comment:
Hmm thinking about it, ideally I think we should add also another "old enough" python version such as 3.8 just to be sure 
In the past we had some issues due to a wrong typehint (!) that broke transformers for python 3.8: https://github.com/TimDettmers/bitsandbytes/pull/859 so I think adding it would circumvents such an issue being on main - what do you think?

Reply:
I just saw https://github.com/TimDettmers/bitsandbytes/issues/1010 I think what you said makes sense, we could do the python=3.8 build in another PR - what do you think?

Reply:
We would add a follow-up step that tests the built libraries on all supported versions, but building the binaries (and wheel) only needs to occur on one version.